docs/custom-flows/add-email.mdx

```diff
@@ -0,0 +1,154 @@
+---
+description: Learn how to use Clerk's API to build a custom flow for adding an email to a user's account.
+---
+
+# Build a custom flow for adding an email to a user's account
+
+<Callout type="danger">
+  This guide is for users who want to build a *custom* user interface using the Clerk API. To use a *prebuilt* UI, you should use Clerk's [Account Portal pages](/docs/account-portal/overview) or [prebuilt components](/docs/components/overview).
+</Callout>
+
+Users are able to add multiple email addresses to their account.
+
+Every user has a [`User`](/docs/references/javascript/user/user) object that represents their account. The `User` object has a `emailAddresses` property that contains all the email addresses associated with the user.
+
+The following example demonstrates how to build a custom user interface that allows users to add an email address to their account. The example:
```

Comment:
It might be worth mentioning that this is a custom version of the `add phone number` function from the `<UserProfile />`